py-polars/tests/unit/test_exprs.py

```diff
@@ -1034,3 +1034,135 @@ def test_is_not_deprecated() -> None:
 
     expected = pl.DataFrame({"a": [False, True, False]})
     assert_frame_equal(result, expected)
+
+
+def test_cut() -> None:
```

Comment:
These tests should go in `tests/unit/series/test_cut.py` (and `qcut`).

Actually they should be under `tests/unit/operations`, but at least they should be together with the other cut/qcut tests.